hyp3_metadata/templates/insar/readme.md.txt.j2

```diff
@@ -0,0 +1,298 @@
+
+A digital elevation model is required for processing InSAR. The DEM used for this particular product is the Copernicus GLO-30 Public DEM, which has a native resolution of 1 arc second (about 30 meters). This DEM is a global Digital Surface Model (DSM) derived from the WorldDEM. The WorldDEM is based on radar satellite data acquired by the TanDEM-X mission, which was funded by the German Aerospace Center (DLR) and Airbus Defence and Space, and edited to flatten water bodies, provide consistent flow of rivers, and apply corrections to shore/coastlines and special features. For an overview of the dataset, visit https://spacedata.copernicus.eu/fr/dataset-details?articleId=394198. The product handbook is available at https://spacedata.copernicus.eu/documents/20126/0/GEO1988-CopernicusDEM-SPE-002_ProductHandbook_I1.00.pdf.
+
+***************
+# Product Contents #
+
+The files generated in this process include:
+
+1. Wrapped Interferogram (PNG images, KMZ file, GeoTIFF (*optional*))
+2. Unwrapped Interferogram (GeoTIFF, PNG images, KMZ file)
+3. Line-of-Sight Displacement Map (GeoTIFF) - *Optional*
+4. Vertical Displacement Map (GeoTIFF)
+5. Coherence Map (GeoTIFF)
+6. Amplitude Image (GeoTIFF)
+7. Parameter Documentation (Text File)
+8. Look Vector maps (GeoTIFFs) - *Optional*
+9. Incidence Angle map (GeoTIFF) - *Optional*
+10. DEM used to process the data (GeoTIFF) - *Optional*
+11. Water Mask (GeoTIFF) - *Optional*
+
+*See below for detailed descriptions of each of the product files.*
+
+----------------
+## 1. Wrapped Interferogram
+
+The wrapped interferogram is output in both a georeferenced PNG and KMZ format by default, and the full GeoTIFF file can also be optionally included. Because the fringes are on a wrapped 2-pi scale rather than representing continuous values, a wrapped interferogram tends to be most useful as a color visualization. This product displays deformation in multiples of half of the sensor wavelength, so areas with fringes that are very close together indicate greater amounts of deformation in the line-of-sight direction from the sensor.
+
+The wrapped interferogram image is generated from the interferogram phase prior to unwrapping. The PNG and KMZ files are tagged with _color_phase, and are 2048 pixels wide. The optional GeoTIFF is tagged with _wrapped_phase.tif
+
+*The GeoTIFF version of the wrapped interferogram is optional. Select the "Include Wrapped GeoTIFF" option in the On Demand HyP3 Processing Options to include it in the product package.*
```

Comment:
I think the proposed label for the option is "Include Wrapped Phase", but I could be talked into "wrapped interferogram" or "wrapped geotiff" if anyone thinks those are better user-facing labels.

Reply:
Tricky. 
Maybe "Include Wrapped Phase GeoTIFF" is a suitable compromise. A bit long, but not too bad. 

The hard part is that the wrapped phase is already included as a browse, which is sufficient (and more useful) for many users. That makes "Include Wrapped Phase" a bit misleading for those who don't actually need the GeoTIFF version, but "Include Wrapped GeoTIFF" reads a bit funny.

Reply:
This has gone to production in Vertex as Include Wrapped Phase, so we'll go with that. It's probably fine anyway, as the color phase browse isn't just a lower-resolution wrapped phase image, but a combination visualization of wrapped phase and backscatter intensity.